analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/SimpleProjectFunctionalTest.java

```diff
@@ -111,6 +111,11 @@ public void ensureAlignmentFileCreated() throws IOException, URISyntaxException,
         assertThat(extraGradleFile).exists();
         assertThat(FileUtils.readLines(extraGradleFile, Charset.defaultCharset()))
                 .filteredOn(l -> l.trim().equals(AlignmentTask.APPLY_GME_REPOS)).hasSize(1);
+
+        // Verify the org.gradle.caching has been removed.
+        File properties = new File(projectRoot, "gradle.properties");
+        assertThat(FileUtils.readLines(properties, Charset.defaultCharset()))
+                .filteredOn(l -> l.contains("org.gradle.caching")).hasSize(0);
```

Comment:
AssertJ does have a quicker/more compact way of handling files, so that you don't have to "manually" read the lines first:

```java
assertThat(linesOf(extraGradleFile)).containsOnlyOnce(AlignmentTask.APPLY_GME_REPOS);
```

```java 
assertThat(linesOf(properties)).doesNotContain("org.gradle.caching");
```

Reply:
Nice! Thanks, have updated the new line.